plugins/sdk.mjs

```diff
@@ -29,12 +29,20 @@ export const makeSdksWithCachedState = async () => {
       let state = AssetService.deserializeState(cachedState.item);
       const provider = providerForChain(chain);
       await provider.ready;
-      const sdk = new Yearn(chain, { provider, cache: { useCache: false } }, state);
+      const sdk = new Yearn(
+        chain,
+        { provider, cache: { useCache: false }, disableAllowlist: true },
+        state
+      );
       sdks[chain] = sdk;
     } else {
       const provider = providerForChain(chain);
       await provider.ready;
-      const sdk = new Yearn(chain, { provider, cache: { useCache: false } });
+      const sdk = new Yearn(chain, {
+        provider,
+        cache: { useCache: false },
+        disableAllowlist: true,
```

Comment:
I'm not sure if this is the right thing to do, but the service won't start up without this

Reply:
Yeah this is correct, we removed this requirement of needing to pass this variable in the latest master of the sdk but it still needs to be released, can leave it like this for now